mindone/env/init.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import os
+from typing import Optional, Tuple
+
+import mindspore as ms
+from mindspore.communication import get_group_size, get_rank, init
+
+from ..utils.seed import set_random_seed
+from .parallel_config import ParallelConfig
+
+_logger = logging.getLogger(__name__)
+
+
+def init_train_env(
+    mode: int = ms.GRAPH_MODE,
+    debug: bool = False,
+    seed: int = 42,
+    cache_graph: bool = False,
+    distributed: bool = False,
+    enable_modelarts: bool = False,
+    num_workers: int = 1,
+    json_data_path: Optional[str] = None,
+) -> Tuple[int, int, int]:
+    """
+    Initialize MindSpore training environment.
+
+    Args:
+        mode: MindSpore execution mode. Default is 0 (ms.GRAPH_MODE).
+        debug: Whether to enable debug mode (forces PyNative mode). Default is False.
```

Comment:
better name as debug -> is_debug_mode

Reply:
This sounds like a function that should return an output, `is_debug_mode() -> True / False`, rather than a flag.